<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-E5NCT7NC" ignored="0" name="Test" tests="10" failures="10" timestamp="2022-04-26T23:17:59 IST" time="93.532" errors="0">
  <testcase name="login" time="5.819" classname="com.bank.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.LoginPageTest.login(LoginPageTest.java:27)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="loginData" time="5.577" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.56" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.516" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.452" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.525" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.509" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.559" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.57" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
  <testcase name="loginData" time="5.621" classname="com.bank.testcases.TC_001_DDTLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [HubSpot Home] but found [HubSpot Login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [HubSpot Home] but found [HubSpot Login]
at com.bank.testcases.TC_001_DDTLogin.loginData(TC_001_DDTLogin.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginData -->
</testsuite> <!-- Test -->
